README.md

```diff
@@ -38,6 +38,8 @@ into `pom.xml` for your project.
 Get your [API keys](https://uploadcare.com/docs/start/settings/#keys) to proceed with
 the examples below.
 
+Read full documentation on [javadoc.io](https://www.javadoc.io/doc/com.uploadcare/uploadcare/latest/index.html).
```

Comment:
this is not full documentation, it's [javadoc](https://en.wikipedia.org/wiki/Javadoc) :)
```suggestion
Read class documentation on [javadoc.io](https://www.javadoc.io/doc/com.uploadcare/uploadcare/latest/index.html).
```